packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -379,6 +379,19 @@ export class Queue<T> implements IStream<T> {
 	}
 }
 
+const waitForOnline = async (): Promise<void> => {
```

Comment:
This looks like adds 3 copies of the same thing in the same loader bundle (I'm actually surprised only 68 bytes added, per measurements published for this PR). Can we avoid duplicating code?

Reply:
I'll see what we can do.

IMO root problem is that both `ConnectionManager` and `ParallelRequests` (and `requestOps()`) should all live hidden in driver layer/package, where they would easily be able to share implementation without public exposure.  `ParallelRequests` and `requestOps()` can fairly easily get moved to a more legitimate driver-layer package (which I'd like to do soon), but not `ConnectionManager`.  So it's a struggle between layer cleanliness, back-compat, and duplicated code.

#14265 will reduce bundle size so I'm paying the debt at least, but I'll see if there's any good clever option.